docs/default.md

```diff
@@ -2,25 +2,13 @@
 
 This is a React + `algob` project which serves as a app template. It has a single package.json file. `algob` works within the project (using `yarn run algob deploy`).
 
-Install `Algo Builder` using
-- `yarn add @algo-builder/algob` or
-- `yarn link @algo-builder/algob` (with local installation. This is recommended if you want to use `algob` with latest, not released version). Read more about it [here](https://github.com/scale-it/algo-builder#installation).
+NOTE: Please go through `setup.md` first, if not visited.
 
 Before running react app with algosigner:
 - Make sure to add your network config to the algosigner extension (display name, id, algod_url, token). Read [this](https://github.com/PureStake/algosigner#custom-networks) for more information on setting up your wallet in algosigner for custom network.
 - Add your ledger name (display name) in `algorand.config.js` for private-net, otherwise it would be "TestNet" or "MainNet"
 - Custom accounts for private-net in `algob.config.js` can be used within `AlgoSigner.algod` API, but to use them with `AlgoSigner.Accounts`, you need to go to the algosigner extension UI -> import existing account (within your private-net configuration) -> add the account mneumonic (say john's in algob.config.js). Then that account will show in your algosigner wallet as well.
```

Comment:
I think this is also common information accross templates. Can we please move this to the setup/intergrations.md as well?

Reply:
sure.